### PR TITLE
typo

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: databroker/tests/test_broker.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def test_uid_list_multiple_headers(db, RE):
     RE.subscribe(db.insert)
     uids = RE(pchain(count([det]), count([det])))
     headers = db[uids]
-    assert uids == [h['start']['uid'] for h in headers]
+    assert uids == tuple([h['start']['uid'] for h in headers])
```
Comment (danielballan) [inline databroker/tests/test_broker.py:67]: I think this change would make the reader wonder why `tuple()` is needed to make the test pass. In fact, of course, it is not needed: lists and tuples may be compared.

Comment (CJ-Wright) [reply databroker/tests/test_broker.py:67]: Please tell me more about this magical comparison.
https://travis-ci.org/NSLS-II/databroker/jobs/269221759#L1455

Comment (danielballan) [reply databroker/tests/test_broker.py:67]: :sheep:. Indeed:

```python
>>> >>> (1,2) == [1,2]
False
```

**Comment by danielballan** [comment]
One Travis build passed; the others are server startup and random network failure. Merging.

**Pull request merged**